### PR TITLE
KCI-9 ✅ Add promtail extraction

### PR DESCRIPTION
**Pull request by deifyed**
## Description

- Adds install script and related resources for Promtail installation

## Motivation and Context

- [KCI-9](https://github.com/oslokommune/kjoremiljo-confidential-issues/issues/9) 
- [Slack thread](https://oslokommune.slack.com/archives/C018EFCSC4W/p1668000724687529)

## How to prove the effect of this PR?

Follow the instructions in the [README.md](./promtail/README.md), then open your Grafana instance, go to explore, then Loki
and ensure logs are being collected.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: gists/helm-component-extraction/promtail/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+## Uninstall okctl provisioned Promtail
+
+1. Edit the relevant `cluster.yaml` file and set `integrations.promtail` to `false`.
+2. Run `okctl apply cluster -f cluster.yaml` to apply the changes.
+
+## Installation
+
+### Install Promtail
+
+1. Place this directory in `infrastructure/<cluster name>/helm-components/`
```
Comment (ivaruf) [inline gists/helm-component-extraction/promtail/README.md:32]: Place what? The values.yaml file?

File: gists/helm-component-extraction/promtail/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+2. **Optional**; edit the `values.yaml` to suit your setup.
+3. Run `make install` to install Promtail.
```
Comment (ivaruf) [inline gists/helm-component-extraction/promtail/README.md:34]: Also copy makefile to same path?

File: gists/helm-component-extraction/promtail/README.md (hunk 1 above)
Comment (ivaruf) [reply gists/helm-component-extraction/promtail/README.md:32]: Make directory if not exists?

File: gists/helm-component-extraction/promtail/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+		```
+2. Log in to the AWS account with `aws sso login`
+
+### Log into okctl
+
+1. Run `okctl venv -c <path to relevant cluster.yaml>` to log into the okctl cluster.
+
+## Uninstall okctl provisioned Promtail
+
+1. Edit the relevant `cluster.yaml` file and set `integrations.promtail` to `false`.
```
Comment (ivaruf) [inline gists/helm-component-extraction/promtail/README.md:25]: Note: Make sure that no integrations are still commented out if you were running on with everything default in your cluster file.

File: gists/helm-component-extraction/promtail/README.md (hunk 1 above)
Comment (deifyed) [reply gists/helm-component-extraction/promtail/README.md:32]: The directory that contains the README this is written in. I.e.: `cp -r <okctl-update-repo-root>/gists/helm-component-extraction/promtail <iac-repo-root>/infrastructure/<cluster name>/helm-components/promtail`

File: gists/helm-component-extraction/promtail/README.md (hunk 2 above)
Comment (deifyed) [reply gists/helm-component-extraction/promtail/README.md:34]: Yeah, the whole directory

**Review by ivaruf — APPROVED** [review]

**Pull request merged**